### PR TITLE
Update how-to-restore-server-portal.md (Typo "flexible server"→"Flexi…

### PR DESCRIPTION
**Pull request by hyoshioka0128**
…ble Server")

https://github.com/MicrosoftDocs/azure-databases-docs/blob/main/articles/mysql/flexible-server/how-to-restore-server-portal.md 
#PingMSFTDocs

### CODE REVIEW

**Comment by hyoshioka0128** [comment]
@code-sidd - related #12

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 1]
```diff
@@ -14,19 +14,19 @@ ms.topic: how-to
 
 [!INCLUDE[applies-to-mysql-flexible-server](../includes/applies-to-mysql-flexible-server.md)]
 
-This article provides step-by-step procedure to perform point-in-time recoveries in Azure Database for MySQL flexible server using backups.
+This article provides step-by-step procedure to perform point-in-time recoveries in Azure Database for MySQL Flexible Server using backups.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:17]:
```suggestion
This article provides step-by-step procedure to perform point-in-time recoveries in Azure Database for MySQL - Flexible Server using backups.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 2]
```diff
@@ -14,19 +14,19 @@ ms.topic: how-to
...
 
 ## Prerequisites
 
 To complete this how-to guide, you need:
 
-- You must have an Azure Database for MySQL flexible server instance.
+- You must have an Azure Database for MySQL Flexible Server instance.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:23]:
```suggestion
- You must have an Azure Database for MySQL - Flexible Server instance.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 3]
```diff
@@ -14,19 +14,19 @@ ms.topic: how-to
...
 
 ## Restore to the latest restore point
 
-Follow these steps to restore your Azure Database for MySQL flexible server instance using an earliest existing backup.
+Follow these steps to restore your Azure Database for MySQL Flexible Server instance using an earliest existing backup.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:27]:
```suggestion
Follow these steps to restore your Azure Database for MySQL - Flexible Server instance using an earliest existing backup.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 4]
```diff
@@ -14,19 +14,19 @@ ms.topic: how-to
...
 
-1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL flexible server instance that you want to restore the backup from.
+1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL Flexible Server instance that you want to restore the backup from.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:29]:
```suggestion
1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL - Flexible Server instance that you want to restore the backup from.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 5]
```diff
@@ -46,9 +46,9 @@ Follow these steps to restore your Azure Database for MySQL flexible server inst
 
 ## Restore to a fastest restore point
 
-Follow these steps to restore your Azure Database for MySQL flexible server instance using an existing full backup as the fastest restore point.
+Follow these steps to restore your Azure Database for MySQL Flexible Server instance using an existing full backup as the fastest restore point.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:49]:
```suggestion
Follow these steps to restore your Azure Database for MySQL - Flexible Server instance using an existing full backup as the fastest restore point.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 6]
```diff
@@ -46,9 +46,9 @@ Follow these steps to restore your Azure Database for MySQL flexible server inst
...
 
-1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL flexible server instance that you want to restore the backup from.
+1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL Flexible Server instance that you want to restore the backup from.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:51]:
```suggestion
1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL - Flexible Server instance that you want to restore the backup from.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 7]
```diff
@@ -70,9 +70,9 @@ Follow these steps to restore your Azure Database for MySQL flexible server inst
 
 ## Restore from a full backup through the Backup and Restore page
 
-Follow these steps to restore your Azure Database for MySQL flexible server instance using an existing full backup.
+Follow these steps to restore your Azure Database for MySQL Flexible Server instance using an existing full backup.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:73]:
```suggestion
Follow these steps to restore your Azure Database for MySQL - Flexible Server instance using an existing full backup.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 8]
```diff
@@ -70,9 +70,9 @@ Follow these steps to restore your Azure Database for MySQL flexible server inst
...
 
-1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL flexible server instance that you want to restore the backup from.
+1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL Flexible Server instance that you want to restore the backup from.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:75]:
```suggestion
1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL - Flexible Server instance that you want to restore the backup from.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 9]
```diff
@@ -92,7 +92,7 @@ Follow these steps to restore your Azure Database for MySQL flexible server inst
 
 ## Geo restore to latest restore point
 
-1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL flexible server instance that you want to restore the backup from.
+1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL Flexible Server instance that you want to restore the backup from.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:95]:
```suggestion
1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL - Flexible Server instance that you want to restore the backup from.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 10]
```diff
@@ -123,9 +123,9 @@ The new server created by geo restore has the same server admin sign-in name and
 
 ## Use restore to move a server from Public access to Private access
 
-Follow these steps to restore your Azure Database for MySQL flexible server instance using an earliest existing backup.
+Follow these steps to restore your Azure Database for MySQL Flexible Server instance using an earliest existing backup.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:126]:
```suggestion
Follow these steps to restore your Azure Database for MySQL - Flexible Server instance using an earliest existing backup.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 11]
```diff
@@ -123,9 +123,9 @@ The new server created by geo restore has the same server admin sign-in name and
...
 
-1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL flexible server instance that you want to restore the backup from.
+1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL Flexible Server instance that you want to restore the backup from.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:128]:
```suggestion
1. In the [Azure portal](https://portal.azure.com/), choose your Azure Database for MySQL - Flexible Server instance that you want to restore the backup from.
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 12]
```diff
@@ -142,18 +142,18 @@ Follow these steps to restore your Azure Database for MySQL flexible server inst
 1. In the **Connectivity method**, select **Private access (VNet Integration)**. Go to **Virtual Network** section, you can either select an already existing *virtual network* and *Subnet* that is delegated to *Microsoft.DBforMySQL/flexibleServers* or Create a new one by selecting the *create virtual network* link.
     > [!NOTE]  
     > Only virtual networks and subnets in the same region and subscription is listed in the dropdown list. </br>
-    > The chosen subnet is delegated to *Microsoft.DBforMySQL/flexibleServers*. It means that only Azure Database for MySQL flexible server instances can use that subnet.</br>
+    > The chosen subnet is delegated to *Microsoft.DBforMySQL/flexibleServers*. It means that only Azure Database for MySQL Flexible Server instances can use that subnet.</br>
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:145]:
```suggestion
    > The chosen subnet is delegated to *Microsoft.DBforMySQL/flexibleServers*. It means that only Azure Database for MySQL - Flexible Server instances can use that subnet.</br>
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 13]
```diff
@@ -142,18 +142,18 @@ Follow these steps to restore your Azure Database for MySQL flexible server inst
...
 
     :::image type="content" source="./media/how-to-manage-virtual-network-portal/vnet-creation.png" alt-text="Screenshot of Vnet configuration." lightbox="./media/how-to-manage-virtual-network-portal/vnet-creation.png":::
 
 1. Create a new or Select an existing **Private DNS Zone**.
     > [!NOTE]  
     > Private DNS zone names must end with `mysql.database.azure.com`. </br>
     > If you don't see the option to create a new private dns zone, enter the server name on the **Basics** tab.</br>
-    > After the Azure Database for MySQL flexible server instance is deployed to a virtual network and subnet, you can't move it to Public access (allowed IP addresses).</br>
+    > After the Azure Database for MySQL Flexible Server instance is deployed to a virtual network and subnet, you can't move it to Public access (allowed IP addresses).</br>
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:153]:
```suggestion
    > After the Azure Database for MySQL - Flexible Server instance is deployed to a virtual network and subnet, you can't move it to Public access (allowed IP addresses).</br>
```

File: articles/mysql/flexible-server/how-to-restore-server-portal.md [hunk 14]
```diff
@@ -142,18 +142,18 @@ Follow these steps to restore your Azure Database for MySQL flexible server inst
...
 
     :::image type="content" source="./media/how-to-manage-virtual-network-portal/private-dns-zone.png" alt-text="Screenshot of dns configuration." lightbox="./media/how-to-manage-virtual-network-portal/private-dns-zone.png":::
-1. Select **Review + create** to review your Azure Database for MySQL flexible server configuration.
+1. Select **Review + create** to review your Azure Database for MySQL Flexible Server configuration.
```
Comment (Court72) [inline articles/mysql/flexible-server/how-to-restore-server-portal.md:156]:
```suggestion
1. Select **Review + create** to review your Azure Database for MySQL - Flexible Server configuration.
```

**Comment by Court72** [comment]
Thanks, @hyshioka0128!

**Pull request merged**